src/services/codefixes/importFixes.ts

```diff
@@ -274,10 +278,20 @@ function createImportAdderWorker(sourceFile: SourceFile | FutureSourceFile, prog
         addImport(first(info));
     }
 
+    function getExportedSymbolParent(symbol: Symbol, checker: TypeChecker): Symbol {

```

Comment:
I wonder if this is something that will only be relevant for import fixes. Could any other code crash similarly due to a missing parent?

Reply:
It would be nice/convenient if the checker set the parent, but I thought it was better not to touch the checker and add a bunch of parent pointers to everyone’s program if `tsc` never needs them. We could try it and see if it shows up in the perf run. At least, we should have a comment here explaining that this function is making up for an arguable checker deficiency.

Reply:
Tried in #59609